configuration/uart.md

```diff
@@ -2,17 +2,15 @@
  
-By default, on Raspberry Pis equipped with the wireless/Bluetooth module (Raspberry Pi 3 and Raspberry Pi Zero W), the PL011 UART is connected to the BT module, while the mini UART is used for Linux console output. On all other models the PL011 is used for the Linux console output. 
+By default, on Raspberry Pis equipped with the wireless/Bluetooth module (Raspberry Pi 3 and Raspberry Pi Zero W), the PL011 UART is connected to the BT module, while the mini UART is used as the primary UART and will have a Linux console on it. On all other models the PL011 is used as the primary UART. 
 
-In Linux device terms, by default, /dev/ttyS0 refers to the mini UART, and /dev/ttyAMA0 refers to the PL011. The primary UART is that assigned to the Linux console, which depends on the Raspberry Pi model as described above, and can be accessed via /dev/serial0.
+In Linux device terms, by default, /dev/ttyS0 refers to the mini UART, and /dev/ttyAMA0 refers to the PL011. The primary UART is that assigned to the Linux console, which depends on the Raspberry Pi model as described above. There are also symlinks /dev/serial0 which always refers to the primary UART (if enabled), and dev/serial1 which similarly always refers to the secondary UART (if enabled).
 
 ## Mini UART and CPU core frequency
 
-The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. Also, when the Linux console uses the mini UART (Raspberry Pi 3, Raspberry Pi Zero W), as a consequence of the UART being disabled, the console is also disabled.
+The baud rate of the mini UART is linked to the core frequency of the VPU on the VC4 GPU. This means that as the VPU frequency governor varies the core frequency, the baud rate of the UART also changes. This makes the UART of limited use in the default state. By default, if the mini UART is selected for use as the primary uart it will be disabled. To enable it, add enable_uart=1 to config.txt. This will also fix the core frequency to 250MHz (unless force_turbo is set, when it will be fixed to 400MHz). When the mini UART is not the primary uart, for example you are using it to connect to the Bluetooth controller, you must add core_freq=250 to config.txt otherwise the mini UART will not work.
```

Comment:
I would rather say "the VPU turbo frequency" than "400MHz". Also "for example you are using" -> "for example if you are using".

Reply:
done